.github/workflows/check-runtimes-template-spec-chain.yml

```diff
@@ -0,0 +1,135 @@
+name: check-runtimes-template-spec-chain.yml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  preflight:
+    uses: ./.github/workflows/reusable-preflight.yml
+
+  build-wasm:
+    runs-on: ${{ needs.preflight.outputs.RUNNER }}
+    needs: preflight
+    container:
+      image: ${{ needs.preflight.outputs.IMAGE }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install srtool-cli
+        run: |
+          cargo install --git https://github.com/chevdor/srtool-cli
+          echo "Using srtool-cli Version:"
+          srtool --version
+
+      - name: Pull srtool docker image
+        run: |
+          srtool pull
```

Comment:
Looks like it can not detect the container engine. I am seeing an error here: https://github.com/paritytech/polkadot-sdk/actions/runs/12244392003/job/34180482077?pr=6529. It is probably because the runner where this runs is not configured with `docker` or `podman`.

Reply:
The job still fails: https://github.com/paritytech/polkadot-sdk/actions/runs/12254440065/job/34185795791?pr=6529. We need to use the `ubuntu-latest` runner for this and drop the reusable preflight runner.